### PR TITLE
Completed html-relative-links.

### PR DESCRIPTION
**Pull request by Youssef-Najjarine**
html-relative-links

### CODE REVIEW

File: html-relative-links/about-me/about-me.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  <title>About Me</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <a href="/html-relative-links/homePage.html"><li>Home Page</li></a>
```
Comment (BrettAlbright) [inline html-relative-links/about-me/about-me.html]: so while this does work, the format you want to go with here is: 
```javascript
href="../index.html"
```

The goal is to route from the current page to the page you want to route to, not to route from the parent directory

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Some updates needed!

File: html-relative-links/about-me/about-me.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <a href="../homePage.html"><li>Home Page</li></a>
+    <a href="/html-relative-links/about-me/about-me.html"><li>About Me</li></a>
```
Comment (BrettAlbright) [inline html-relative-links/about-me/about-me.html]: well done on the update! now you just need to do it for the rest of them! Make sure to update all links as needed!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Moving in the right direction!

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**